app/letters/utils.py

```diff
@@ -25,8 +25,8 @@ class ScanErrorType(Enum):
 PRECOMPILED_BUCKET_PREFIX = '{folder}NOTIFY.{reference}'
 
 
-def get_folder_name(_now, is_test_or_scan_letter=False):
-    if is_test_or_scan_letter:
+def get_folder_name(_now, dont_use_sending_date=False):
```

Comment:
i'd much rather this variable was a positive rather than a negative (eg `use_sending_date=True`), that's how most booleans are used.

Reply:
decision: lets not worry about this in this PR